ProcessesApi/V1/Services/ChangeOfNameService.cs

```diff
@@ -42,8 +44,46 @@ public void AddAppointmentDateTimeToEvent(Stateless.StateMachine<string, string>
             _eventData = ProcessHelper.CreateEventData(trigger.FormData, new List<string> { SharedKeys.AppointmentDateTime });
         }
 
+        private async Task OnProcessClosed(Stateless.StateMachine<string, string>.Transition x)
+        {
+            var processRequest = x.Parameters[0] as ProcessTrigger;
+            _eventData = ProcessHelper.ValidateHasNotifiedResident(processRequest);
+            await PublishProcessClosedEvent(x).ConfigureAwait(false);
+        }
+
+        private async Task OnProcessCancelled(Stateless.StateMachine<string, string>.Transition x)
+        {
+            var processRequest = x.Parameters[0] as ProcessTrigger;
+            ProcessHelper.ValidateFormData(processRequest.FormData, new List<string>() { SharedKeys.Comment });
+
+            _eventData = ProcessHelper.CreateEventData(processRequest.FormData, new List<string> { SharedKeys.Comment });
+            await PublishProcessClosedEvent(x).ConfigureAwait(false);
+        }
+
+        private async Task ReviewDocumentsCheck(StateMachine<string, string>.Transition transition)
+        {
+            var processRequest = transition.Parameters[0] as ProcessTrigger;
+            var formData = processRequest.FormData;
+
+            var expectedFormDataKeys = new List<string>{
+                  SharedKeys.SeenPhotographicId,
+                  SharedKeys.SeenSecondId,
+                  ChangeOfNameKeys.AtLeastOneDocument,
+            };
+            ProcessHelper.ValidateFormData(formData, expectedFormDataKeys);
```

Comment:
Shouldn't you be using ValidateManualCheck here? And make sure all values are true?